internal/tools/aws/database_migration.go

```diff
@@ -0,0 +1,163 @@
+package aws
+
+import (
+	"github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/aws/awserr"
+	"github.com/aws/aws-sdk-go/service/ec2"
+	"github.com/aws/aws-sdk-go/service/rds"
+	"github.com/pkg/errors"
+	log "github.com/sirupsen/logrus"
+
+	"github.com/mattermost/mattermost-cloud/model"
+)
+
+// RDSDatabaseMigration is a migrated database backed by AWS RDS.
+type RDSDatabaseMigration struct {
+	awsClient            *Client
+	masterInstallationID string
+	slaveInstallationID  string
+}
+
+// NewRDSDatabaseMigration returns a new RDSDatabaseMigration.
+func NewRDSDatabaseMigration(masterInstallationID, slaveInstallationID string, awsClient *Client) *RDSDatabaseMigration {
+	return &RDSDatabaseMigration{
+		awsClient:            awsClient,
+		masterInstallationID: masterInstallationID,
+		slaveInstallationID:  slaveInstallationID,
+	}
+}
+
+// Setup sets access from one RDS database to another and sets any configuration needed for replication.
+func (d *RDSDatabaseMigration) Setup(logger log.FieldLogger) (string, error) {
+	masterInstanceSG, err := d.describeDBInstanceSecurityGroup(RDSMasterInstanceID(d.masterInstallationID))
+	if err != nil {
+		return "", errors.Wrapf(err, "unable to setup database migration for installation id: %s", d.masterInstallationID)
+	}
+
+	slaveInstanceSG, err := d.describeDBInstanceSecurityGroup(RDSMigrationInstanceID(d.slaveInstallationID))
+	if err != nil {
+		return "", errors.Wrapf(err, "unable to setup database migration for installation id: %s", d.masterInstallationID)
```

Comment:
Should the error include that the slaveInstance was unable to be found? 0/5 on this.

Reply:
` slaveInstance ID was unable to be found` will probably be available in the wrapped error, but what we were trying migrate was the master installation. I think the error message could contain more information though.